src/platform/Kconfig

```diff
@@ -227,6 +226,7 @@ config MAX_CORE_COUNT
 
 config CORE_COUNT
 	int "Number of cores"
+	default MP_NUM_CPUS if KERNEL_BIN_NAME = "zephyr"
```

Comment:
Btw, are we aligned with all the Zephyr Kconfig symbols ?

Reply:
@lgirdwood sorry, what exactly do you mean by "aligned?" We still have a few duplicating Kconfig symbols, there has been no progress on that. But at least I'm trying to synchronise all the multicore related symbols - number of cores (used), multicore support on or off, with these patches you shouldn't be getting any meaningless (or at least confusing) configurations like SMP=y MULTICORE=n etc.